pkg/api/adjustment.go

```diff
@@ -147,6 +147,24 @@ func (a *ContainerAdjustment) AddCDIDevice(d *CDIDevice) {
 	a.CDIDevices = append(a.CDIDevices, d) // TODO: should we dup d here ?
 }
 
+// SetLinuxMemoryPolicyMode records setting the memory policy mode for a container.
+func (a *ContainerAdjustment) SetLinuxMemoryPolicyMode(mode string) {
+	a.initLinuxMemoryPolicy()
+	a.Linux.MemoryPolicy.Mode = mode
+}
+
+// SetLinuxMemoryPolicyNodes records setting memory policy nodes for a container.
+func (a *ContainerAdjustment) SetLinuxMemoryPolicyNodes(nodes string) {
+	a.initLinuxMemoryPolicy()
+	a.Linux.MemoryPolicy.Nodes = nodes
+}
+
+// SetLinuxMemoryPolicyFlags records setting memory policy flags for a container.
+func (a *ContainerAdjustment) SetLinuxMemoryPolicyFlags(flags []string) {
+	a.initLinuxMemoryPolicy()
+	a.Linux.MemoryPolicy.Flags = flags
+}
+
```

Comment:
If the dominant usage pattern of this is going to be calling at least both `SetLinuxMemoryPolicy{Mode,Nodes}()`, then we could also consider adding a `func (a *ContainerAdjustment) SetLinuxMemoryPolicy(mode, nodes string, flags ...string)`. (Or, in reference to my other review comment, using enum types instead of strings if we end up opting for that...)

Reply:
Ended up separating all three (Mode, Nodes, Flags) into separate functions because MPOL_DEFAULT and MPOL_LOCAL require that Nodes must be not be defined. Or more exactly, the nodemask must be 0.

Of course, this does not prevent having `SetLinuxMemoryPolicy(mode, nodes string, flags ...string)` where nodes just must be empty string.

For type safety, I'd actually like to use SetLinuxMemoryPolicy(MemoryPolicyModeType, string, ...MemoryPolicyFlagType) here, but as this api does not import runtime-spec, so it would look a bit strange to have it for this case only.

Anyway, I like your "flags... string" better than "flags []string".

Let's discuss tomorrow face-to-face what would be the best combo out of all these.